Docker.DotNet/IContainerOperations.cs

```diff
@@ -20,6 +20,8 @@ public interface IContainerOperations
 
         Task<Stream> ExportContainerAsync(string id, CancellationToken cancellationToken);
 
+        Task<ExecContainerResponse> ExecContainerAsync(string id, ExecContainerParameters parameters);
```

Comment:
thanks for updating the PR. I realized there are a lot of Exec operations (ExecCreate, ExecStart, ExecResize etc...) also ExecContainer is not the right terminology. Can we rename this param (and ...Parameters type) to `ContainerExecCreate` please?